<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerseEntryWdgt</class>
 <widget class="QWidget" name="VerseEntryWdgt">
  <property name="windowTitle">
   <string>Verse Entry Widget</string>
  </property>
  <layout class="QVBoxLayout" name="VLVerseEntryWdgt">
   <item>
    <layout class="QHBoxLayout" name="HLLocation">
     <item>
      <widget class="QLabel" name="LBibleBook">
       <property name="text">
        <string>Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBBibleBook"/>
     </item>
     <item>
      <widget class="QLabel" name="LChapterNo">
       <property name="text">
        <string>Chapter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBChapterNo">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LVerseNo">
       <property name="text">
        <string>Verse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBVerseNo">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>176</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="PTEVerseText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
